core/src/openedx/org/openedx/core/ui/theme/Colors.kt

```diff
@@ -10,24 +10,37 @@ val light_background = Color.White
 val light_surface = Color(0xFFF7F7F8)
 val light_error = Color(0xFFFF3D71)
 val light_onPrimary = Color.White
-val light_onSecondary = Color.Black
+val light_onSecondary = Color.White
 val light_onBackground = Color.Black
 val light_onSurface = Color.Black
 val light_onError = Color.White
+val light_onWarning = Color.White
+val light_onInfo = Color.White
+val light_info_variant = Color(0xFF3C68FF)
 val light_text_primary = Color(0xFF212121)
 val light_text_primary_variant = Color(0xFF3D4964)
+val light_text_primary_light = light_text_primary
 val light_text_secondary = Color(0xFFB3B3B3)
 val light_text_dark = Color(0xFF19212F)
 val light_text_accent = Color(0xFF3C68FF)
-val light_text_warning= Color(0xFF19212F)
+val light_text_warning = Color(0xFF19212F)
 val light_text_field_background = Color(0xFFF7F7F8)
 val light_text_field_background_variant = Color.White
 val light_text_field_border = Color(0xFF97A5BB)
 val light_text_field_text = Color(0xFF3D4964)
 val light_text_field_hint = Color(0xFF97A5BB)
-val light_button_background = Color(0xFF3C68FF)
-val light_button_secondary_background = Color(0xFF79889F)
-val light_button_text = Color.White
+val light_text_hyper_link = Color(0xFF3C68FF)
+
+val light_primary_button_background = Color(0xFF3C68FF)
+val light_primary_button_border = Color(0xFF97A5BB)
+val light_primary_button_text = Color.White
+val light_primary_button_bordered_text = Color(0xFF3C68FF)
+val light_secondary_button_background = Color(0xFF3C68FF)
+val light_secondary_button_text = Color.White
+val light_secondary_button_border = Color(0xFF97A5BB)
+val light_secondary_button_bordered_background = Color.White
+val light_secondary_button_bordered_text = Color(0xFF3C68FF)
```

Comment:
Could we reuse them this way?
```suggestion
val light_primary_button_background = Color(0xFF3C68FF)
val light_primary_button_border = Color(0xFF97A5BB)
val light_primary_button_text = Color.White
val light_primary_button_bordered_text = Color(0xFF3C68FF)

val light_secondary_button_background = light_primary_button_background
val light_secondary_button_text = light_primary_button_text
val light_secondary_button_border = light_primary_button_border
val light_secondary_button_bordered_background = Color.White
val light_secondary_button_bordered_text = light_primary_button_bordered_text
```

Reply:
addressed.